Description: Use the newer lcms2 in debian 
Author: Matthias Geiger <werdahias@riseup.net> 
Forwarded: https://gitlab.gnome.org/sophie-h/glycin/-/merge_requests/16
Last-Update: 2023-09-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/src/icc.rs
+++ b/src/icc.rs
@@ -46,7 +46,7 @@ pub fn apply_transformation(frame: &Fram
     }
 }
 
-fn transform<F: Copy>(
+fn transform<F: Copy + rgb::Pod>(
     icc_profile: &[u8],
     memory_format: MemoryFormat,
     buf: &mut [F],
